src/parkapi_sources/converters/herrenberg_bike/models.py

```diff
@@ -0,0 +1,156 @@
+"""
+Copyright 2024 binary butterfly GmbH
+Use of this source code is governed by an MIT-style license that can be found in the LICENSE.txt.
+"""
+
+from datetime import date, datetime, time, timezone
+from enum import Enum
+from typing import Optional
+
+from validataclass.dataclasses import DefaultUnset, ValidataclassMixin, validataclass
+from validataclass.helpers import OptionalUnset, UnsetValue
+from validataclass.validators import (
+    DataclassValidator,
+    EnumValidator,
+    IntegerValidator,
+    Noneable,
+    NoneToUnsetValue,
+    StringValidator,
+    UrlValidator,
+)
+
+from parkapi_sources.converters.base_converter.pull import GeojsonFeatureGeometryInput
+from parkapi_sources.models import StaticParkingSiteInput
+from parkapi_sources.models.enums import ParkingSiteType, PurposeType, SupervisionType
+from parkapi_sources.validators import MappedBooleanValidator, ParsedDateValidator
+
+
+class HerrenbergBikeType(Enum):
+    STANDS = 'stands'
+    WALL_LOOPS = 'wall_loops'
+    RACK = 'rack'
+    SHED = 'shed'
+    BOLLARD = 'bollard'
+    WIDE_STANDS = 'wide_stands'
+    BUILDING = 'building'
+    LOCKERS = 'lockers'
+    WAVE = 'wave'
+    ANCHORS = 'anchors'
+    FLOOR = 'floor'
+    SAFE_LOOPS = 'safe_loops'
+    GROUND_SLOTS = 'ground_slots'
+    LEAN_AND_STICK = 'lean_and_stick'
+    CROSSBAR = 'crossbar'
+    OTHER = 'other'
+
+    def to_parking_site_type(self) -> ParkingSiteType:
+        if self in [self.OTHER, self.BOLLARD, self.LEAN_AND_STICK, self.WAVE, self.ANCHORS, self.CROSSBAR, self.RACK, self.GROUND_SLOTS]:
+            return ParkingSiteType.OTHER
+        if self == self.SAFE_LOOPS:
```

Comment:
Why not using the dict below for these single checks?

Reply:
Now done